neo/io/blackrockio.py

```diff
@@ -1765,7 +1766,7 @@ def __read_spiketrain(
             wf_size = self.__nev_params('waveform_size')[channel_idx]
 
             waveforms = spikes['waveform'].flatten().view(wf_dtype)
-            waveforms = waveforms.reshape(spikes.size, 1, wf_size)
```

Comment:
depending on your exact version of python2.7 this will either give the following warning or error out completely:
```
neo/io/blackrockio.py:1769: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  waveforms = waveforms.reshape(spikes.size, 1, wf_size)
```